### ISSUE TITLE
Internal Server Error?

### ISSUE THREAD

**Issue by AmazingEric** [peterhajas/LivingDex#11]
Hi, I encounter an error when I sign up. Have a screenshot:

![image](https://cloud.githubusercontent.com/assets/8645405/12994081/04c2729a-d0ec-11e5-9263-4657a9eb5441.png)

I am using the Torch browser. I have also tried it on my iPhone, with the same result.

**Comment by ShannonKleiner** [comment]
I'm getting the same error on attempted registration in Chrome 48.0.2564.116 m.

![ise](https://cloud.githubusercontent.com/assets/4149797/13228703/3ae93836-d952-11e5-9ef3-732876d16b1d.png)

**Comment by NormalGamerGal** [comment]
Attempted registration in Chrome, Opera, Firefox, Safari, and multiple IE versions and got the same error as above. I'm on Windows 7. Would be nice to know if anyone's working on this, or if I should just create my own system for my living dex. Thanks!

**Comment by peterhajas** [comment]
Yep, this is a server side issue.

Unfortunately, I only got the free Heroku plan. As a result, the app is unable to create additional rows for new users. I could buy more, but the app has some database issues (poor tracking of forms, limits to the storage per Pokemon, etc.)

I'll be looking in to fixing this, along with some other database scaling problems that prohibited things like Shiny tracking. I want LivingDex to be useful for Sun and Moon, also.

I'll keep this issue open until I fix the issue.

**Comment by AmazingEric** [comment]
Okay. Sounds like a plan. :)

**Comment by ShannonKleiner** [comment]
Thanks for the update, Peter!

**Comment by NormalGamerGal** [comment]
Yes, thank you for responding! I'm a web developer, so I understand your problem, and how annoying/costly space can be. Hope it gets resolved eventually, but until then, I'll try and figure out something on my own. Again, thanks for the update!

**Comment by OrangeCrush** [comment]
Thanks for the work on this peter!

**Comment by basvroegop** [comment]
Came here right after running into the issue, glad to see it's being worked on! Looking forward to using your website. Goodluck.

**Comment by nko89** [comment]
Is there anymore update on this error?

**Comment by xstefanx** [comment]
Just wanted to check in again to see if this is still being worked on. I'd love to use the site, but it looks as if I'm late to the party. 

On a side note, I was going to try and install it in windows, but I'm not sure that's an option >.<

**Comment by NiCKCoRY** [comment]
@peterhajas Just wondering if you had a little more information on this issue. I really want to use the site as It looks incredible, but sadly can't.

**Comment by M0UL** [comment]
any update on this?

**Comment by area** [comment]
I'd suggest using http://pokedextracker.com in the meantime.

**Comment by AmazingEric** [comment]
Thanks, area.

Again, there wasn't enough space for his web plan. The issue will be closed when it's fixed. :)
